### ISSUE TITLE
GPL

### ISSUE THREAD

**Issue by handicraftsman** [aseprite/aseprite#1280]
Can you publish non-commercical gpl-powered version?

**Comment by dacap** [comment]
Hi @handicraftsman, I think a GPL with that restriction isn't GPL anymore.

**Comment by handicraftsman** [comment]
Check Qt if want to see this in action

**Comment by dacap** [comment]
Qt uses unmodified GPL/LGPL licenses. They offer the commercial license just in case your program cannot meet the GPL/LGPL requirements. See section 3.5 in their FAQ: https://www.qt.io/faq/#_Toc453700714

In our case the "use" of the program wouldn't be completely free anymore with the "non-commercial" restriction. So this GPL part:

> This License explicitly affirms your unlimited permission to run the unmodified Program.

wouldn't be true anymore. Because you shouldn't be able to run the program for commercial purposes anymore. So this would make the software non-free/non-GPL/or with a modified-GPL license.

So basically Qt creates a dual-license model giving the commercial license choice to remove all GPL restrictions to Qt users (the developers). But we cannot do the same because our "commercial license" should start with a modified/restricted GPL in the first place for end-users (the main difference here is that our users are not compiling/linking our code with their creations, that are sprites/image files).
